source/whats-new.txt

```diff
@@ -224,6 +251,9 @@ New features of the 5.5 {+driver-short+} release include:
   later if you want to create an encrypted collection by using the
   Queryable Encryption feature.
 
+To learn more about this release, see the `v5.5.0 Release Highlights
+<https://github.com/mongodb/node-mongodb-native/releases/tag/v5.5.0>`__.
+
```

Comment:
Suggestion:
No action necessary, but older changes might be better in a separate PR to make it easier to backport to relevant versions. I think this page is one that the team should segment by major version, so it's probably not worthwhile backporting.